cypress/fixtures/constants.js

```diff
@@ -0,0 +1,141 @@
+export const TEST_DOCUMENT = {
```

Comment:
Should this file not have the open source header comment?

Reply:
That is correct.  fixing now.